### PR TITLE
Added SSL support

### PR DESCRIPTION
**Pull request by carlitoselmago**
fix for "getusermedia no longer works on insecure origins"

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,11 +6,13 @@
 import os
 import signal
 import sys
+import ssl
 
 from aiohttp import errors, web
 from jsonmodels import models, fields
 from jsonmodels.errors import ValidationError
 
+port=5000
```
Comment (saghul) [inline app.py]: please don't do this.  Either make it properly configurable or leave it as it was.

File: app.py [hunk 2]
```diff
@@ -32,8 +37,10 @@ def __init__(self, filename, content_type):
     def __call__(self, request):
         if self.data is None:
             try:
+
```
Comment (saghul) [inline app.py]: unneeded change

File: app.py [hunk 3]
```diff
@@ -32,8 +37,10 @@ def __init__(self, filename, content_type):
     def __call__(self, request):
         if self.data is None:
             try:
+                
                 with open(self.filename, 'rb') as f:
                     self.data = f.read()
+
```
Comment (saghul) [inline app.py]: unneeded change

File: app.py [hunk 4]
```diff
@@ -155,7 +162,7 @@ def __init__(self):
 
     @asyncio.coroutine
     def __call__(self, request):
-        ws = web.WebSocketResponse(protocols=('callroulette-v2',))
+        ws = web.WebSocketResponse(protocols=('callroulette-v2'))
```
Comment (saghul) [inline app.py]: it's a tuple, the comma is relevant.

File: app.py [hunk 5]
```diff
@@ -262,14 +269,17 @@ def parse(data):
 
 @asyncio.coroutine
 def init(loop):
+
```
Comment (saghul) [inline app.py]: unneeded change

File: app.py [hunk 6]
```diff
@@ -262,14 +269,17 @@ def parse(data):
...
     app = web.Application(loop=loop)
     app.router.add_route('GET', '/', LazyFileHandler(INDEX_FILE, 'text/html'))
     app.router.add_route('GET', '/ws', WebSocketHandler())
     app.router.add_route('GET', '/static/{path:.*}', StaticFilesHandler(STATIC_FILES))
 
     handler = app.make_handler()
-    server = yield from loop.create_server(handler, '0.0.0.0', 8080)
-    print("Server started at 0.0.0.0:8080")
+    server = yield from loop.create_server(handler, '0.0.0.0', port,ssl=sslcontext)
```
Comment (saghul) [inline app.py]: style: space after comma

File: static/js/app.js [hunk 7]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (saghul) [inline static/js/app.js:1]: please leave this file unformatted, I don't like unrelated style changes. Except the wss change, of course :-)

**Review by saghul — CHANGES_REQUESTED** [review]
Thanks for doing this!  Left some comments.

**Comment by carlitoselmago** [comment]
I reverted some of the non related format changes etc. Now it's just the strict fix, nothing else.

**Comment by saghul** [comment]
Great work, thanks!

**Pull request merged**